lms/djangoapps/instructor/tests/test_api.py

```diff
@@ -498,7 +503,7 @@ def _access_problem_responses_endpoint(self, msg):
         mock_problem_key.course_key = self.course.id
         with patch.object(UsageKey, 'from_string') as patched_method:
             patched_method.return_value = mock_problem_key
-            self._access_endpoint('get_problem_responses', {}, 200, msg)
+            self._access_endpoint(endpoint, {}, 200, msg)
```

Comment:
I've reused the existing tests here, and just pointed them to run against the new API as well.